src/versions/develop/pages/FO/hummingbird/category/index.ts

```diff
@@ -40,7 +40,7 @@ class CategoryPage extends CategoryPageVersion implements FoCategoryPageInterfac
     this.searchFiltersLabel = `${this.searchFilters} label.form-check-label`;
     this.clearAllFiltersLink = `${this.searchFilters} button.js-search-filters-clear-all`;
     this.searchFilterPriceSlider = 'div.faceted-slider';
-    this.searchFiltersSlider = 'div.noUi-base';
+    this.searchFiltersSlider = () => `${this.searchFilters} div.faceted-slider`;
```

Comment:
```suggestion
    this.searchFiltersSlider =  `${this.searchFilters} div.faceted-slider`;
```

Reply:
Not in this case. Why ? Because the parent class has parameters to this property.... And we don't use it... so we declare with `()` but in empty mode.